mountpoint-s3/src/cli.rs

```diff
@@ -667,7 +666,6 @@ pub fn create_s3_client(args: &CliArgs) -> anyhow::Result<(S3CrtClient, EventLoo
         .read_backpressure(true)
         .initial_read_window(initial_read_window_size)
         .user_agent(user_agent);
-    #[cfg(feature = "multiple-nw-iface")]
```

Comment:
I've completely overlooked that on previous review. Is it possible to verify locally that it has the desired effect though? (e.g. compiling without any feature flags and specifying `eth0` and an inexistent interface as in your auto tests)

(also there's a conflict in changelog)

Reply:
Yep, I verified locally (and double checked just now after merging from main).